ocaml/libs/tracing/tracing.ml

```diff
@@ -78,6 +80,12 @@ module Spans = struct
             if List.length span_list < 1000 then
               Hashtbl.replace spans key (span :: span_list)
     )
+
+  let since () =
```

Comment:
This should have a doc-comment explaining why it clears the hashtable

Reply:
Added